src/clojurewerkz/elastisch/rest/document.clj

```diff
@@ -264,9 +264,10 @@
 
 (defn scroll-seq
   "Returns a lazy sequence of all documents for a given scroll query"
-  ([^Connection conn prev-resp {:keys [search_type] scroll-ttl :scroll
-                                :or {scroll "1m"} :as opts}]
```

Comment:
To be honest, neither did I until I did a bit of REPL experimentation as a result of looking over this PR ;)